rs/nervous_system/integration_tests/tests/sns_release_qualification.rs

```diff
@@ -296,3 +302,100 @@ async fn test_sns_upgrade(sns_canisters_to_upgrade: Vec<SnsCanisterType>) {
         .await;
     }
 }
+
+// DO NOT MERGE
```

Comment:
?

Reply:
This test should be removed before we merge this PR, as the normal qualification tests would cover this scenario. 

For context, this PR is blocked on https://github.com/dfinity/ic/pull/2427